torchtext/csrc/regex.h

```diff
@@ -0,0 +1,12 @@
+#include <string>
+#include <torch/script.h>
```

Comment:
Are you using this one for pybind2 benchmark?

Reply:
I have a separate branch for pybind11 that I will create a PR with once this is merged in. Thats the branch I used to get the pybind11 numbers